app/lib/services.rb

```diff
@@ -16,4 +16,8 @@ def self.content_store
       disable_cache: true
     )
   end
+
+  def self.draft_origin
+    @draft_origin ||= Plek.new.external_url_for('draft-origin')
```

Comment:
This feels a little weird.  The other services are things to interact with rather than pieces of configuration.  I'd be happier with the inline Plek call in `preview_url` tbh.

Reply:
The next set of changes in #428 removes the need for this.